### DISCUSSION TITLE
Add Timezone Converter

### DISCUSSION THREAD

**Discussion by Ruedi1000** [CorentinTh/it-tools#426] [Ideas]
Hi

I know the Date-time converter on it-tools. Thats great. But we are interesting on Time Zone Converter. That would me shot the time from another timezones.

Thank you for your great work and support.

**Comment by Hallsie** [comment]
I agree with this one.  Was just trying to figure out UTC to EST so I could note some maintenance windows for a SAAS application.

**Reply by JDProfresh** [reply]
> I agree with this one. Was just trying to figure out UTC to EST so I could note some maintenance windows for a SAAS application.

Issue #429 was created for this idea

**Comment by sharevb** [comment]
Hi, for people interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Comment by sharevb** [comment]
Hi @Ruedi1000 and @JDProfresh, implemented here : https://sharevb-it-tools.vercel.app/timezone-converter
